README.md

```diff
@@ -1 +1,44 @@
-Authenticator Sample App
+# Authenticator sample app
+
+This sample application demonstrates an authenticator-only application that uses PingOne for Customers mobile SDK. An iOS developer can easily build a branded and customized Authenticator application using this sample.
+
+## What’s in the sample app?
+
+  - Full native application written in Swift 5.1, compatible for all iOS devices from iOS 10 or above.
+  - Integration with PingOne SDK version 1.2.0.
+  - UI customization can be done easily to get your company flavor on the app.
+  - All app texts can be easily localized and modified in one file.
+
+## Features
+
+  - Pairing flow using QR code with camera scan or manual entry.
+  - Override usernames locally, or use names from PingOne directory.
+  - Authentication flow with push notifications using biometric FaceID and TouchID.
+  - Side menu with send logs option to track customer issues with support ID.
+
+## Prerequisites
+
+The Authenticator sample app requires Xcode 11 and above to compile and run.
+
+To set up your application for  working with push messages in iOS refer to [PingOne mobile SDK iOS readMe].
+
+## Installation
+
+1. Download the latest code [here].
```

Comment:
the link should point to **master** branch

Reply:
Fixed